piccolo_admin/endpoints.py

```diff
@@ -388,6 +392,20 @@ def __init__(
             ),
         )
 
+        auth_app.mount(
+            path="/change-password/",
+            app=AuthenticationMiddleware(
+                change_password(login_url=f"{self.prefix_path}auth/login/"),
+                SessionsAuthBackend(),
+            ),
+        )
+
+        auth_app.add_route(
+            path="/register/",
+            route=register(redirect_to=f"{self.prefix_path}auth/login/"),
+            methods=["POST"],
+        )
```

Comment:
We need to be really careful about this - we don't want anybody to be able to register. I think this needs to be mounted to the `api_app`, so it requires authentication. Then only existing admin users can register new users.

Reply:
Now I have to go, but I will try to solve it during the day.

Reply:
No worries - catch you later.